### PR TITLE
Information about Sonic Generations' Unused HUD Animations

### PR DESCRIPTION
**Pull request by TheExentist151**
Information about Unused HUD Animtions for Time Eater Boss

### CODE REVIEW

**Comment by PTKay** [comment]
For assets, please create a folder with the same name as the document and put the assets there.
Example:

```
assets/document/file.gif
```
document being the name of the .md file

File: docs/discoveries/bb/hudanims/blb_hudanims.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+description: Info about Unused HUD Animations in Sonic Generations
```
Comment (PTKay) [inline docs/discoveries/bb/hudanims/blb_hudanims.md]: Change "Unused HUD Animations" to "unused HUD animations"

File: docs/discoveries/bb/hudanims/blb_hudanims.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+There's some unused HUD animations for Time Eater in <code style="color:red;">bb.cpk/SonicActionCommonHud.ar.00\ui_gameplay.xncp</code>.
```
Comment (PTKay) [inline docs/discoveries/bb/hudanims/blb_hudanims.md:5]: This file is missing a title. Please specify a title using `#` in the markdown file.

Example:

```
---
description: Info about unused HUD animations in Sonic Generations
---

# Unused HUD Animations
There's some (...)
```

**Comment by TheExentist151** [comment]
Updated

**Review by PTKay — CHANGES_REQUESTED** [review]
I'm afraid the assets still aren't properly organised. Considering these files are all used in the `blb_hudanims` .md file, set them up like this:

```
hudanims/assets/blb_hudanims/change_c2g.gif
hudanims/assets/blb_hudanims/change_c2w.gif
(...) and so on
```
Update the usages in the `blb_hudanimd.md` file accordingly.

Also, there's should not be .md files in `assets`.

File: docs/discoveries/bb/hudanims/blb_hudanims.md (hunk 2 above)
Comment (TheExentist151) [reply docs/discoveries/bb/hudanims/blb_hudanims.md:5]: Updated again

**Comment by PTKay** [comment]
Looks good now! Thanks for the effort 👍

**Review by PTKay — APPROVED** [review]

**Pull request merged**